bigint/src/lib.rs

```diff
@@ -127,6 +127,7 @@ pub mod big_digit {
 
     // `DoubleBigDigit` size dependent
     pub const BITS: usize = 32;
+    pub use ::std::u32::MAX;
```

Comment:
Let's not add any new `pub` items please.  In a perfect world, I wouldn't even make `big_digit` public at all...
Just make it a top-level private name if you need to use it throughout the module.